<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookInputForm</class>
 <widget class="QDialog" name="BookInputForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>201</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить книгу</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Имя автора:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Название:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Жанр:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Тип:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Издательство:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Количество:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
